js/holmes.js

```diff
@@ -86,42 +86,45 @@
      * Options
      * @type {Object}
      */
-    holmes.prototype.options = options;
+    holmes.prototype.options = {
+      input: 'input[type=search]',
+      placeholder: '',
+      placeholderNode: null,
+      mark: false,
+      class: {
+        visible: '',
+        hidden: 'hidden'
+      },
+      dynamic: false,
+      minCharacters: 0,
+      find: ''
+    };
+
 
     // if holmes.prototype.options.find is missing, the searching won't work so we'll thrown an exceptions
     if (typeof holmes.prototype.options.find !== 'string') {
       throw new Error('A find argument is needed. That should be a querySelectorAll for each of the items you want to match individually. You should have something like: \nholmes({\n\tfind:".result"\n});\nsee also https://haroen.me/holmes/doc/module-holmes.html');
     }
 
+    holmes.prototype.mergeObj = function (Obj1, Obj2) {
```

Comment:
Oh yeah good point, that shouldn't have been there. The `Obj1` should then say something like "object that gets Obj2 injected" or something like that.